### DISCUSSION TITLE
Documentation on filter chain

### DISCUSSION THREAD

**Discussion by jandermoreira** [quarto-dev/quarto-cli#10016] [Q&A] [answered]
### Description

Where can I find the documentation on filter chains to get details and also how to use this information in custom Lua filters?

I refer to #7066.

**Comment by mcanouil** [comment, marked as answer]
See:

- https://quarto.org/docs/extensions/filters.html
- https://quarto.org/docs/prerelease/1.4/lua_changes.html#more-precise-targeting-of-ast-processing-phases (not yet part of the main documentation page)

**Comment by jandermoreira** [comment]
Great! I'll take a look.
